src/main/java/frc/robot/RobotContainer.java

```diff
@@ -162,21 +165,47 @@ public void initializeAutoBuilder() {
   }
 
   public void registerCommands() {
-    NamedCommands.registerCommand("Intake_Note_1.5_Sec", new IntakeNote(intake, index, arm).withTimeout(1)); // 3 seconds used at SVR
+    NamedCommands.registerCommand("Intake_Note_1.5_Sec", new IntakeNote(intake, index).withTimeout(1)); // 3 seconds used at SVR
+    NamedCommands.registerCommand("Stop", new RunCommand(()->drive.stop()).withTimeout(1)); 
     // NamedCommands.registerCommand("Stop", new WaitCommand(2)); // to represent stopping for shooting 
     // Mehaan -- Consulted with Jerry, just going to use a constraint zone going at .1 which should be fine instead of stopping for the area in which we are supposed to shoot
     // NamedCommands.registerCommand("PrepareShooter", new PrepareShooter(shooter, 0));
     // NamedCommands.registerCommand("SetShooterSpeed", new SetShooterSpeed(shooter, 0));
     // NamedCommands.registerCommand("ShootKnownPos", new ShootKnownPos(shooter, arm, index, null));
-    NamedCommands.registerCommand("Outtake_Note_1.5_Sec", new SequentialCommandGroup(
-      new ParallelDeadlineGroup(// TODO: This will end instantly
+    
+    //Original
+    NamedCommands.registerCommand("Outtake_Note_1.5_Sec", new SequentialCommandGroup(// TODO: This will end instantly
+    // TODO: Don't use setChassisSpeeds(), use drive() instead and add the drivetrain as a parameter so it is a requirement
+      new ParallelDeadlineGroup(new PrepareShooter(shooter, 1750),
+      new WaitCommand(.75)),
+      new WaitCommand(.75),
+      new InstantCommand(()-> index.runIndex()),
+      new WaitCommand(.75),
+      new ParallelDeadlineGroup(new PrepareShooter(shooter, 0))
```

Comment:
Why do you have a deadline group with 1 thing in it? Also, why have a parallel command group with a prepare shooter and wait command with a wait command after it? Make either one of those wait commands twice as long and remove the other one.

Reply:
Same thing for the command below this one.